pkg/kubelet/pod/pod_manager.go

```diff
@@ -80,7 +80,7 @@ type Manager interface {
 	// DeletePod deletes the given pod from the manager.  For mirror pods,
```

Comment:
`s/DeletePod/RemovePod`

Reply:
We could potentially fix this in the followup PR I mentioned above too.